boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -689,6 +705,9 @@ LoadWindowsCore(IN USHORT OperatingSystemVersion,
         NoexecuteDisabled = TRUE;
     }
 
+    PaeModeOn = WinLdrIsPaeSupported(KernelFileName, HalFileName);
+    FIXME("WinLdrIsPaeSupported: PaeModeOn %X\n", PaeModeOn);
```

Comment:
Trace? or Err?

Reply:
This is a FIXME because the execution order needs to be changed here. First parse boot menu, then test PAE and depending on PaeModeOn load "ntoskrnl.exe" (or "ntkrnlpa.exe") and download the rest. That is, you need the boot menu at the very beginning, and the download of files at the end.
PS: I suggested this in another PR...